frontend/device/kindle/device.lua

```diff
@@ -989,7 +989,10 @@ function KindleBasic3:init()
 
     Kindle.init(self)
 
-    self.input.snow_protocol = true -- cf. https://github.com/koreader/koreader/issues/5070
+    -- This device doesn't emit ABS_MT_TRACKING_ID:-1 events on contact lift,
+    -- so we have to rely on contact lift detection via BTN_TOUCH:0,
+    -- c.f., https://github.com/koreader/koreader/issues/5070
+    self.input.snow_protocol = true
```

Comment:
A Kindle with Snow protocol, teehee.

Reply:
Thank's for the fun. I read your word, I believe I can follow you to some point, but at then I see BAHNHOF. :) :)

For the non-German speaking (per google translate from https://www.geo.de/geolino/redewendungen/9327-rtkl-ich-verstehe-immer-nur-bahnhof):
Idiom: Understand the train station (German: BAHNHOF)

The saying comes from the end of the First World War. At that time, the soldiers were very tired after the long battles and just wanted to go home. And we went home from the train station. So they connected the idea of the train station with the journey home.

Then, when someone asked them something, they would reply, "I only ever understand train station" - which meant they didn't want to talk about anything other than the journey home. And when they were asked about other topics, they couldn't say anything more because they didn't know anything about the subject and only understood "train station".

We still use this phrase today when we don't understand anything at all - or when we want to reject a conversation.